### ISSUE TITLE
jQuery 3 support

### ISSUE THREAD

**Issue by adeelejaz** [volmer/bootsy#258]
When using Bootsy with jQuery 3, I get the following warning and error:

Warning:
```
jQuery.Deferred exception: url.indexOf is not a function TypeError: url.indexOf is not a function
```

Error:
```
Uncaught TypeError: url.indexOf is not a function
```

I tracked the issue down jQuery dropping support for ```.load()```: [Breaking change: .load(), .unload(), and .error() removed](https://jquery.com/upgrade-guide/3.0/#breaking-change-load-unload-and-error-removed)

The issue is in [init.js#L30](https://github.com/volmer/bootsy/blob/master/app/assets/javascripts/bootsy/init.js#L30):

```js
/* Initialize Bootsy on document load */
$(function() {
  $(window).load(function() {
    Bootsy.init();

    /* Reload Bootsy on page load when using Turbolinks. */
    document.addEventListener('turbolinks:load', Bootsy.init);
  });
});
```

In jQuery 3.0:
```js
$(window).load(function() {
```
has to be changed to
```js
$(window).on('load', function() {
```

Alternatively, reverting backback to jQuery 2 fixes the issue.

**Comment by seanfcarroll** [comment]
Im getting the same error

**Comment by volmer** [comment]
Fixed at #259
